texttospeechv1/text_to_speech_v1_adapter.go

```diff
@@ -0,0 +1,141 @@
+/**
+ * (C) Copyright IBM Corp. 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package texttospeechv1
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/IBM/go-sdk-core/core"
+	common "github.com/watson-developer-cloud/go-sdk/common"
+)
+
+type SynthesizeCallbackWrapper interface {
+	OnOpen()
+	OnError(error)
+	// OnContentType(contentType)
+	OnTimingInformation(timings Timings)
+	// OnAudioStream(audioStream)
+	OnData(*core.DetailedResponse)
+	OnClose()
```

Comment:
is this tabs?

Reply:
Yes

Reply:
https://golang.org/cmd/gofmt/ this is the formatting

Reply:
Looks like it is influenced by vscode, in vscode it looks like this:
<img width="1574" alt="Screen Shot 2019-09-10 at 2 58 22 PM" src="https://user-images.githubusercontent.com/13935994/64653507-74a85c00-d3db-11e9-9137-a33d90650fde.png">

Reply:
In Go, the recommended indent level is a tab character and is enforced by the gofmt tool I think.